CONTRIBUTING.md

```diff
@@ -100,28 +112,34 @@ Follow the instructions below on how to install Bundler and setup the database.
   * Download latest stable: `wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb`
   * Install chrome: `sudo dpkg -i google-chrome-stable_current_amd64.deb`
 
-#### Getting the code
-
-* Clone the repo: `git clone git://github.com/rubygems/rubygems.org`
-* Move into your cloned rubygems directory if you haven’t already:
-    `cd rubygems.org`
```

Comment:
I moved this to the top of the setup instructions since steps above this rely on already having the repository cloned.